drive/driveimpl/drive_test.go

```diff
@@ -154,7 +154,7 @@ func newSystem(t *testing.T) *system {
 	// Make sure we don't leak goroutines
 	tstest.ResourceCheck(t)
 
-	fs := NewFileSystemForLocal(log.Printf)
+	fs := newFileSystemForLocal(log.Printf, nil)
```

Comment:
The unit tests were only passing because stat caching was broken. With stat caching on, the unit tests fail.